### ISSUE TITLE
SPEC stages / decision points / language

### ISSUE THREAD

**Issue by jarrodmillman** [scientific-python/specs#84]
This is a follow-up to issue #81.

The language about the stages / decision points is not great.  Currently, this is what we have

![screenshot](https://user-images.githubusercontent.com/123428/110721739-f1c9b280-81c5-11eb-9fc8-192f0754a133.png)

In #79, there are these options

![screenshot](https://user-images.githubusercontent.com/123428/110721818-1cb40680-81c6-11eb-8f24-7c026c07b7bd.png)

Option 1 is most similar to the current draft where the focus is on decision points.  Options 2 and 3 are more related to my older idea of organizing things into stages during which each SPEC document is in a certain state and "ends" with a decision made by some group about the SPEC document.

**Comment by dschult** [comment]
I like Option 3. :}  It makes it more clear who makes the decision, and the stages are identified with minimal overlap with the verbs that describe the decisions.

**Comment by dschult** [comment]
The big thing is to get the language decided for both states and actions. Then the rest is already written and pretty clear.  One suggestion is to reduce the number of names by calling the stages: Proposal, Draft SPEC and SPEC. Then the verbs are clearly verbs and not stages. 

We propose the proposal. The Steering Committee Accepts the proposal and turns it into a Draft SPEC. The Core Projects endorse the Draft SPEC. and the Steering Committee (or an administrator?) approves it when 2 Core Projects endorse it. Approval involves changing it from a Draft SPEC to a SPEC. Further endorsements can be made. But if the number of endorsements falls below 2, the SPEC is no longer Approved and becomes a Draft SPEC.

**what a mess**...  OK... so I think the verbs were: propose, accept, endorse, approve, adopt.

**Comment by jarrodmillman** [comment]
As soon as two core projects are listed in the `endorsed-by` field of a SPEC in the main branch, the SPEC is no longer marked as a draft:  https://github.com/scientific-python/scientific-python.org/commit/26860228015530a23b1e089a72d627268113862c

**Comment by pllim** [comment]
Once this is finalized, would be nice to have the process clearly laid out in repo README so it is immediately clear to outsiders.

For instance, a random repo wants to start adopting some of these SPECs and realized they also maybe want to propose an update or a completely new SPEC. But their repo is not listed in your `core-projects`. Should they even bother? Who are the SPECs for? How do you make it into `core-projects`? Can you be part of this without being part of `core-projects`?

**Comment by bsipocz** [comment]
> Can you be part of this without being part of core-projects?

yes, there are the non domain specific core-projects. They endorse and adopt specs, but other libraries can also adopt them.
